awd/cli.py

```diff
@@ -35,24 +37,24 @@ def create_list_of_dspace_item_files(
 
 
 def doi_to_be_added(doi: str, doi_items: list[dict[str, Any]]) -> bool:
-    "Validate that a DOI is not a part of the database table and needs to  be added."
+    """Validate that a DOI is not a part of the database table and needs to  be added."""
     validation_status = False
     if not any(doi_item["doi"] == doi for doi_item in doi_items):
         validation_status = True
-        logger.debug(f"{doi} added to database.")
```

Comment:
I can go either way, I've internalized the no-f-strings-in-log-messages rule

Reply:
Spot checked a couple projects, and it looks like they all align with the "no-f-strings-in-log-messages" rule.  I'm fine with that too.  If we _keep_ this rule in place, then it would enforce that throughout the projects when they have `ruff` and our configurations applied.  

I know I said I'd be okay ignoring this rule above, but for consistency and less unneeded changes in other projects, I might vote to keep this rule in place.